akvo/rsr/static/scripts-src/my-results/components/Indicators.jsx

```diff
@@ -29,34 +32,41 @@ import {
 import Periods from './Periods';
 
 
-const IndicatorHeader = ({indicator, aggregateActualValue}) => {
+const IndicatorHeader = ({indicator, aggregateActualValue, aggregateCompletionPercentage}) => {
     const title = indicator.title.length > 0 ? indicator.title : _("nameless_indicator");
+    // Don't show progress bar if there's no target value (aggregateCompletionPercentage is NaN)
+    const progress_bar = aggregateCompletionPercentage !== aggregateCompletionPercentage ? undefined : (
+        // Limit percentage to 100 for progress bar to work correctly
+        <li><Progress percent={Math.min(100, aggregateCompletionPercentage)} /></li>
+    )
     return (
         <span className="indicatorTitle">
             <ul>
                 <li>{title}</li>
                 <li><span className="aggrActualValue">
                     {_("aggregate_actual_value")}<span>{aggregateActualValue}</span></span>
                 </li>
-            </ul>   
+                {progress_bar}
+            </ul>
         </span>
     )
 };
 IndicatorHeader.propTypes = {
     indicator: PropTypes.object,
     aggregateActualValue: PropTypes.number,
+    aggregateCompletionPercentage: PropTypes.number,
 };
 
 
 const IndicatorContent = ({indicator}) => {
     const description = indicator.description.length > 0 ?
-        <ul>
-            <li className="description">
-                <span>{indicator.description}</span>
-            </li>
-        </ul>
-    :
-        undefined;
+                        <ul>
+                            <li className="description">
+                                <span>{indicator.description}</span>
+                            </li>
+                        </ul>
+              :
```

Comment:
Heh, this poor colon is kinda hangin' all by itself 😉 When I write multiline ternary statements I see everything up to and including ? as the if: and the : as the else: and indent as if it's python 😛